### PR TITLE
Doc - Add guide by Frank about Single Instance vs. Cluster

### PR DESCRIPTION
**Pull request by Simran-B**
Does it really explain the difference between _Single Instance_ and _Cluster_ only, or does _Single Instance_ include Active Failover? If so, then we need to change the name to something else, because we use _Single Instance_ as term for a single arangod process without fail over, replication or anything.

Can not test before https://github.com/arangodb/oskar/pull/64 is merged and the Docker docu image is updated.

### CODE REVIEW

File: Documentation/Books/Manual/SUMMARY.md [hunk 1]
```diff
@@ -301,6 +302,7 @@
     * [AWS](Deployment/Cloud/AWS.md)
     * [Azure](Deployment/Cloud/Azure.md)
   * [Production Checklist](Deployment/ProductionChecklist.md)
+  * [Migrating from a _Single Instance_ to a _Cluster_](Deployment/MigratingSingleInstanceCluster.md)
```
Comment (Simran-B) [inline Documentation/Books/Manual/SUMMARY.md]: Not a fan of using italicized text in the navigation to be honest. We should either not add it here, or change everywhere in the SUMMARY.

File: Documentation/Books/Manual/Deployment/MigratingSingleInstanceCluster.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+section
+{% endhint %}
+
+To migrate from a _Single Instance_ to a _Cluster_ you will need
+to take a backup from the _Single Instance_ and restore it into
+the _Cluster_. The tools [_arangodump_](../Programs/Arangodump/README.md)
+and [_arangorestore_](../Programs/Arangorestore/README.md) can be used
+for such scope.
+
+After that, the _arangod_ server should be restarted with the desired storage
```
[the remark this answers is not in the document]
Comment (sleto-it) [reply Documentation/Books/Manual/Deployment/MigratingSingleInstanceCluster.md]: I am not sure from where this entire "After that.." sentence come from. Was not my intention to add it - removing it.

File: Documentation/Books/Manual/Deployment/MigratingSingleInstanceCluster.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+Migrating from a _Single Instance_ to a _Cluster_
+==================================================
+
+{% hint 'warning' %}
+Before migrating from a _Single Instance_ to a Cluster,
+please make sure to read the 
+["Single Instance vs. Cluster"](../Architecture/DifferenceSingleCluster.md)
```
[the remark this answers is not in the document]
Comment (sleto-it) [reply Documentation/Books/Manual/Deployment/MigratingSingleInstanceCluster.md]: done

File: Documentation/Books/Manual/Deployment/MigratingSingleInstanceCluster.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+section
+{% endhint %}
+
+To migrate from a _Single Instance_ to a _Cluster_ you will need
+to take a backup from the _Single Instance_ and restore it into
+the _Cluster_. The tools [_arangodump_](../Programs/Arangodump/README.md)
+and [_arangorestore_](../Programs/Arangorestore/README.md) can be used
+for such scope.
```
[the remark this answers is not in the document]
Comment (sleto-it) [reply Documentation/Books/Manual/Deployment/MigratingSingleInstanceCluster.md]: pls suggest a different wording.. thanks!

File: Documentation/Books/Manual/SUMMARY.md (hunk 1 above)
Comment (sleto-it) [reply Documentation/Books/Manual/SUMMARY.md]: done

File: Documentation/Books/Manual/Deployment/MigratingSingleInstanceCluster.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (Simran-B) [reply Documentation/Books/Manual/Deployment/MigratingSingleInstanceCluster.md]: Simply make it once sentence I would say:
```markdown
To migrate from a _Single Instance_ to a _Cluster_ you will need
to take a backup from the _Single Instance_ and restore it into
the _Cluster_ with the tools [_arangodump_](../Programs/Arangodump/README.md)
and [_arangorestore_](../Programs/Arangorestore/README.md).
```

**Comment by sleto-it** [comment]
http://jenkins01.arangodb.biz:8080/job/arangodb-documentation/197/

**Comment by sleto-it** [comment]
http://jenkins01.arangodb.biz:8080/job/arangodb-documentation/198/

**Comment by sleto-it** [comment]
http://jenkins01.arangodb.biz:8080/job/arangodb-documentation/199/

**Comment by sleto-it** [comment]
merged - when possible, @Simran-B please backport. Thanks,

**Pull request merged**